net/developer-guide/presentation-content/manage-text/manage-bullet-and-numbered-lists/_index.md

```diff
@@ -0,0 +1,135 @@
+---
+title: Manage Bullet and Numbered Lists
+type: docs
+weight: 60
+url: /net/manage-bullet-and-numbered-lists
+
+---
+
+In **Microsoft PowerPoint**, you can create bullet and numbered lists the same way you do in Word and other text editors. **Aspose.Slides for .NET** also allows you to use bullets and numbers in slides in your presentations. 
+
+### Why Use Bullet Lists?
+
+Bullet lists help you to organize and present information quickly and efficiently. 
+
+**Bullet List Example**
+
+In most cases, a bullet list serves these three main functions:
+
+- draws your readers or viewers attention to important information
+- allows your readers or viewers to scan for key points easily
+- communicates and delivers important details efficiently.
+
+### Why Use Numbered Lists?
+
+Numbered lists also help in organizing and presenting information. Ideally, you should use numbers (in place of bullets) when the order of the entries (for example, *step 1, step 2*, etc.) is important or when an entry has to be referenced (for example, *see step 3*).
+
+**Numbered List Example**
+
+This is a summary of the steps (1-15) in the **Creating Bullets** procedure below:
+
+1. Create an instance of the presentation class. 
+2. Perform several tasks (step 3 to step 15).
+3. Save the presentation. 
+
+## Creating Bullets 
+
+To create a bullet list, through these steps:
+
+1. Create an instance of the Presentation class.
+2. Access the slide (in which you want to add a bullet list) in slide collection through the ISlide object.
+3. Add an AutoShape in the selected slide.
+4. Access the TextFrame of the added shape.
+5. Remove the default paragraph in the TextFrame.
+6. Create the first paragraph instance using Paragraph class.
+7. Set the bullet type of the paragraph.
+8. Set the bullet type to Symbol and then set the bullet character.
```

Comment:
Step 7 duplicates step 8 and can be deleted.

Automatically ordered lists are supported by Github, so I think it will be convenient to use them:
https://guides.github.com/features/mastering-markdown/